templates/rundeck-config.properties.j2

```diff
@@ -16,6 +16,11 @@ rundeck.log4j.config.file = /etc/rundeck/log4j.properties
 # Ldap settings
 rundeck.security.syncLdapUser=true
 
+# #File Upload Plugin
+rundeck.fileupload.plugin.type=[provider]
```

Comment:
I'm not sure about its provider.  Do you have any idea?
I've found it  `rundeck.fileupload.plugin.type=filesystem-temp` in the website https://github.com/rundeck/rundeck/issues/4369